webapp/components/NavBar/NavBar.spec.js

```diff
@@ -0,0 +1,86 @@
+import { createLocalVue, shallowMount, RouterLinkStub } from "@vue/test-utils";
+import Vuex from "vuex";
+import NavBar from "./NavBar.vue";
+
+const localVue = createLocalVue();
+localVue.use(Vuex);
+
+describe("NavBar.vue", () => {
+    let wrapper;
+    let actions;
+    let getters;
+    let store;
+
+    const createComponent = () => {
+        store = new Vuex.Store({
+            actions,
+            getters,
+        });
+        wrapper = shallowMount(NavBar, {
+            store,
+            localVue,
+            stubs: {
+                NuxtLink: RouterLinkStub,
+            },
+        });
+    };
+    beforeEach(() => {
+        getters = {
+            loggedIn: () => false,
+        };
+    });
+
+    afterEach(() => {
+        wrapper.destroy();
+    });
+
+    it("renders a Vue component", () => {
+        createComponent();
+        expect(wrapper.exists()).toBe(true);
```

Comment:
I prefer to have my setup methods to return the test subject:
```suggestion
        const wrapper = createComponent();
        expect(wrapper.exists()).toBe(true);
```